Readme.md

```diff
@@ -108,33 +108,49 @@ Starting 30 September 2025, authentication using API keys is no longer supported
 
 #### Setting up Entra Authentication for Application Insights
 
+> ⚠ This section is required if you have disabled `Local Authentication` on your AI resource, See [the docs](https://learn.microsoft.com/en-us/azure/azure-monitor/app/azure-ad-authentication?WT.mc_id=Portal-AppInsightsExtension&tabs=net#disable-local-authentication).
+
+If you want to use Entra Authentication for Application Insights, or if you have disabled `Local Authentication` on your AI resource, you will have to set up the `EntraAuthenticationType` option to the authentication type you want to use (`ManagedIdentity` or `ServicePrincipal`) in the `Lombiq_Hosting_Azure_ApplicationInsights` section of your configuration like below.
```

Comment:
Even after disabling Local Auth with `"EntraAuthenticationType": "None"` telemetry is being ingested for me just with a connstring, including Live Metrics working (not telemetry filtering, that needs the secure control channel). So, it perhaps only affects the API key? Please check.

Reply:
I have also noticed this behavior before which just caused more confusion, however it doesn't seem to be working consistently and is also not in line with Microsofts own docs.

Perhaps this is due to a grace period or something but as per the docs, what is currently described in the readme is the correct way going forward, so I would suggest keeping it like this. Let me know what you think

Reply:
> While testing the setup, I clarified the docs directly instead of doing a dozen code suggestions, but please check.

I am not exactly sure what you mean by this

Reply:
OK then!

Just that I pushed to the repo.

Reply:
Ingesting telemetry with local auth disabled still works for me after 3 days... But whatever, it really shouldn't, nothing to do.